penny_u_django/home/templates/home/base.html

```diff
@@ -11,6 +11,13 @@
     <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
           integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
 
+    {% load static %}
+    <link rel="stylesheet" type="text/css" href="{% static 'home/style.css' %}">
+    <link rel="shortcut icon" type="image/x-icon" href="{% static 'home/favicon.ico' %}">
+    <link rel="icon" type="image/x-icon" href="{% static 'home/favicon.ico' %}">
+
+    <!-- Add icon library -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```

Comment:
Font Awesome is... AWESOME!

Reply:
For speed I think you can also build a custom font awesome library that will only load the ones you need. It makes it slightly faster.